### PR TITLE
impliment update votes on articles

### PR DESCRIPTION
**Pull request by laurie60**

### CODE REVIEW

File: db/controller.js [hunk 1]
```diff
@@ -21,9 +18,20 @@ exports.getArticleById = (req, res, next) => {
...
 
-exports.updateVotes = (req, res) => {
+exports.updateVotes = (req, res, next) => {
   console.log(req.body);
-  alterVotes(req.body).then((alteredArticle) => {
-    res.send(200).send({ alteredArticle }).catch(next);
-  });
+  //console.log(!req.body.inc_votes, "<<<<<<<<!req.body.inc_votes");
+
+  if (!req.body.inc_votes) {
```
Comment (P-Copley) [inline db/controller.js]:
```js
  if (!req.body.inc_votes) {
```
This includes all falsey values. Notably 0 is falsey so your api would send a message of `"invalid input type"` if you sent `inc_votes: 0`

File: __tests__/app.test.js [hunk 2]
```diff
@@ -76,25 +76,63 @@ describe("News Express App", () => {
...
+  describe("PATCH /api/articles/:article_id", () => {
+    test("Should alter votes of secified article by given amount", () => {
+      const changeVotes = { inc_votes: -100 };
+      return request(app)
+        .patch("/api/articles/1")
+        .send(changeVotes)
+        .expect(200)
+        .then(({ body }) => {
+          console.log(body, "<<<in test");
+          expect(body.alteredArticle).toEqual(
```
Comment (P-Copley) [inline __tests__/app.test.js]:
```suggestion
          expect(body.article).toEqual(
```
Keep the keys on your response objects consistent. This should be `article` as that's the resource it represents

**Comment by P-Copley** [comment]
Nice looking endpoint @laurie60

There's a few console.logs and comments that have made their way into this PR. Give it a tidy up and take a look at the comments above before merging 👍

File: db/controller.js (hunk 1 above)
Comment (laurie60) [reply db/controller.js]: Altered code to:   
```if (!req.body.inc_votes && req.body.inc_votes !== 0) ```

**Pull request merged**